guide/popular-topics/intents.md

```diff
@@ -32,24 +32,23 @@ Please make sure to provide the list of gateway intents and partials you use in
 
 ## The Intents Bitfield
 
-discord.js provides the utility structure <docs-link path="class/Intents">`Intents`</docs-link> to simplify the modification of intents bitfields.
+discord.js provides the utility structure <docs-link path="class/IntentsBitField">`IntentsBitField`</docs-link> to simplify the modification of intents bitfields.
 
-You can use the `.add()` and `.remove()` methods to add or remove flags (Intents string literals representing a certain bit) and modify the bitfield. You can provide single flags as well as an array or bitfield. To use a set of intents as a template you can pass it to the constructor. Note that the empty constructor `new Intents()` creates an empty Intents instance, representing no intents or the bitfield `0`:
+You can use the `.add()` and `.remove()` methods to add or remove flags (Intents string literals representing a certain bit) and modify the bitfield. You can provide single flags as well as an array or bitfield. To use a set of intents as a template you can pass it to the constructor. Note that the empty constructor `new IntentsBitField()` creates an empty Intents instance, representing no intents or the bitfield `0`:
 
 ```js
-const { Client, Intents } = require('discord.js');
+const { Client, GatewayIntentBits, IntentsBitField } = require('discord.js');
 
 const myIntents = new Intents();
-myIntents.add(Intents.FLAGS.GUILD_PRESENCES, Intents.FLAGS.GUILD_MEMBERS);
+myIntents.add(GatewayIntentBits.GuildPresences, GatewayIntentBits.GuildMembers);
 
 const client = new Client({ intents: myIntents });
 
 // other examples:
+const otherIntents = new IntentsBitField([GatewayIntentBits.Guilds, GatewayIntentBits.DirectMessages]);
+otherIntents.remove([GatewayIntentBits.DirectMessages]);
 
-const otherIntents = new Intents([Intents.FLAGS.GUILDS, Intents.FLAGS.DIRECT_MESSAGES]);
-otherIntents.remove([Intents.FLAGS.DIRECT_MESSAGES]);
-
-const otherIntents2 = new Intents(32509);
+const otherIntents2 = new IntentsBitField(32509);
```

Comment:
`GatewayIntentBits` shouldn't be used here at all. This is to show the usage of `IntentsBitField`.